python/pyproject.toml

```diff
@@ -13,17 +13,55 @@
 # limitations under the License.
 
 [build-system]
-
+build-backend = "setuptools.build_meta"
 requires = [
     "wheel",
     "setuptools",
     "cython>=0.29,<0.30",
     "scikit-build>=0.13.1",
     "cmake>=3.23.1",
     "ninja",
-    "cuda-python>=11.7.1,<12.0"
+    "cuda-python>=11.7.1,<12.0",
```

Comment:
Shall we alphabetize this list?

Reply:
Just tacked them on to the end to keep the diff easy to read. No strong opinions

Reply:
Let's defer on this; the plan is to start autogenerating these contents from dependencies.yaml once this PR is merged anyway.